sdk/spring/azure-spring-data-cosmos/src/main/java/com/azure/spring/data/cosmos/repository/query/CosmosParameters.java

```diff
@@ -31,8 +31,4 @@ protected CosmosParameters createFrom(List<CosmosParameter> parameters) {
         return new CosmosParameters(parameters);
     }
 
-    @Override
```

Comment:
Will this be considered as a breaking change? Will this work with lower version?